libs/blocks/aside/aside.css

```diff
@@ -93,23 +122,44 @@
   margin-bottom: 0;
 }
 
-.aside .foreground.container .text .action-area>a {
+.aside .foreground.container .text .action-area > a {
   margin-right: var(--spacing-s);
 }
 
+.aside .foreground.container .text .action-area > a:last-child {
+  margin-right: 0;
+}
+
+.aside .foreground.container .text .action-area > a:not(.con-button) {
+  text-decoration: underline;
```

Comment:
We need global link underline styles in typography.css. I'd shy away from this deep of specificity.

Reply:
Again probably another task to sweep cleanup global link styles.

Reply:
okay removed this one